app/src/main/java/com/kickstarter/ui/intentmappers/ProjectIntentMapper.kt

```diff
@@ -90,6 +92,30 @@ object ProjectIntentMapper {
             .mergeWith(projectFromParceledParam)
     }
 
+    /**
+     * Returns an observable of projects retrieved from intent data. May hit the API if the intent only contains a project
+     * param rather than a parceled project.
+     */
+    fun project(intent: Intent, client: ApiClientTypeV2): io.reactivex.Observable<Project> {
```

Comment:
could we add the test for V2  on `ProjectIntentMapperTest`

Reply:
you got it!